lib/PrintAsClang/compatibility_symbols

```diff
@@ -0,0 +1,38 @@
+IBSegueAction
```

Comment:
Nit: These are not symbols in the usual sense, since they aren’t linkable. “Identifier”? “Macro?”

Reply:
They're "symbols" in the sense that they would be emitted in a symbol graph and appear in documentation, though. "Symbols for documentation" has a looser definition than "symbols for linking".